hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMMXBean.java

```diff
@@ -74,6 +75,8 @@ public interface SCMMXBean extends ServiceRuntimeInfo {
 
   String getScmRatisRoles();
 
+  List<List<String>> getRatisRoles();
```

Comment:
This is a good idea! I will work on improving the code. Do you think it would be feasible to remove the original `getScmRatisRoles` and rename `getRatisRoles` to `getScmRatisRoles`?

Reply:
Thanks for the update. The latest change is fine.